sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -331,7 +334,16 @@ private static float setDefaultConfidenceValue(Float confidence) {
      */
     private static List<FormField> toFieldValueArray(List<FieldValue> valueArray, List<ReadResult> readResults) {
         return valueArray.stream()
-            .map(fieldValue -> setFormField(null, null, fieldValue, readResults))
+            .map(fieldValue -> {
+                FieldData valueData = null;
+                // ARRAY has ho value data, such as bounding box.
+                if (ARRAY != fieldValue.getType()) {
```

Comment:
We don't need this check as whenever the fields value type == ARRAY it will always be set to null.
We could keep the comment for future reference.

Reply:
if we don't have this array type check, the valueData won't be null. We pass this valueData into setFormField(), which directly apply the valueData to    
`FormField(name, null, valueData, value, setDefaultConfidenceValue(fieldValue.getConfidence()));`